assets/js/atomic/blocks/product-elements/rating/edit.tsx

```diff
@@ -40,11 +43,4 @@ const Edit = ( {
 		</div>
 	);
 };
-export default withProductSelector( {
-	icon: BLOCK_ICON,
-	label: BLOCK_TITLE,
-	description: __(
-		'Choose a product to display its rating.',
```

Comment:
Are we OK with removing this string? It looks like we're now using the `BLOCK_DESCRIPTION` from `constants.tsx`. If this is ok then ignore this comment! :)

Reply:
Thanks for pointing this out, @opr. I assume this part comes from a time when a merchant could use this component as a standalone component. Since product elements can no longer be accessed directly, this description does not appear anywhere. To avoid confusion, I removed `BLOCK_DESCRIPTION`  from this file.